docs/src/components/document-renderer.tsx

```diff
@@ -111,4 +113,18 @@ const componentBlockRenderers: InferRenderersForComponentBlocks<
       </div>
     );
   },
+  'cloud-image': ({ href: src, alt, sizes, height, width }) => {
```

Comment:
The Keystatic field has `srcSet` as an option. Should we use that here if provided, or remove it from the UI config?

Reply:
Yes, we should. I'm not sure if you want to sync on feature requirements now or later for this but I just threw a few things in I thought would be useful. I also haven't used `next/image` for reasons. If you're happy to just get something going now, maybe we can sync later?

Reply:
Let's keep it simple for now as there is so much design and tech work we need to do before we know what this will actually look like. This will do nicely for now.